docs/developer/gift-cards.mdx

```diff
@@ -463,29 +226,44 @@ mutation {
 }
 ```
 
-As a result, we get the expiry settings updated.
-
-```graphql
-{
-  "data": {
-    "giftCardSettingsUpdate": {
-      "giftCardSettings": {
-        "expiryType": "EXPIRY_PERIOD",
-        "expiryPeriod": {
-          "type": "YEAR",
-          "amount": 12
-        }
-      }
-    }
-  }
-}
-```
+## Using Gift Cards in Checkout
+
+### Applying Gift Cards
+To use a gift card in checkout, run the [`checkoutAddPromoCode`](/api-reference/checkout/mutations/checkout-add-promo-code) mutation with the gift card code.
+
+:::important
+- The channel currency must match the gift card currency
+- If the card was previously used, the checkout user must match the gift card user
+- The card can be used multiple times until the balance is depleted
+:::
+
+### Tax Handling
+Gift cards are applied to the checkout total after tax calculation:
+1. First, Saleor calculates all taxes for the order
+2. Then, the gift card amount is subtracted from the gross total price
+3. The net amount and tax amount are adjusted proportionally
+
+### Multiple Gift Cards
+When multiple gift cards are used:
+- Saleor will always try to use the full amount from each gift card
+- The system processes gift cards in the order they were added to the checkout
+- If the checkout total is less than the gift card amount, only the necessary portion will be used
+- The remaining balance stays on the gift card for future use
+
+### Balance Changes
+Gift card balances are only updated after the order is completed:
+- During checkout, the `currentBalance` remains unchanged
+- After order completion, the used amount is deducted from the gift card's balance
```

Comment:
I've read about checkout today and got confused if it meant checkout or card balance 😓. Maybe it could be rewritten, for example:

- Adding card to the checkout does not modify the card balance
- Used amount will be deducted during checkout completion